src/test/uImanpulate.test.js

```diff
@@ -0,0 +1,46 @@
+import { JSDOM } from 'jsdom';
+import addItemUI, { clear, diplayTask } from '../__mocks__/uImanpulate.js';
+
+const dom = new JSDOM(`<!DOCTYPE html><body><input type="text" name="add-task" id="add-item"  ><div class="list"></div></body>`);// eslint-disable-line
+
+global.document = dom.window.document;
+global.window = dom.window;
+const list = document.querySelector('.list');
+const task = {
+  description: 'Eat',
+  completed: false,
+  index: 0,
+};
+const data = [
+  {
+    description: 'Wake Up',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Sleep',
+    completed: false,
+    index: 2,
+  },
+];
+
+describe('tests for UI ', () => {
+  describe('Test for Ui', () => {
+    test('Add item to list', () => {
+      addItemUI(task);
+      expect(list.childElementCount).toBe(1);
+    });
+  });
```

Comment:
- [ ] It seems you've failed to add tests for the following functionalities, kindly add them:
- Add test for editing a to-do item.
- Add test to update completed status of a to-do item.
- Add test for Clear all completed functionality.

Reply:
Edit function has been tested addRemove.test.js.

Reply:
Thank you